.github/scripts/run-zombienet-test.sh

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env bash
+
```

Comment:
Would be nice to have some context here too.

Reply:
Done